docs/specs/metadata.yml

```diff
@@ -429,3 +429,63 @@ outputs:
         <meta content=\"8\" name=\"wordCount\" />"
     }
   docs/a.mta.json:
+---
+# output defined xrefs per file to structure json
+repos:
```

Comment:
Update all these new tests to use `inputs` to speed up tests

Reply:
the repos used here is for providing the branch name which will be used to construct absolute URL in xref href.
https://github.com/freewheel70/docfx/blob/ad3176f1383257b054b5094fb7379e2d734f253a/src/docfx/build/xref/XrefResolver.cs#L230

Reply:
The branch name should already been tested by some other tests dedicated for testing URLs. A test should focus on one small aspect to be readable.